### ISSUE TITLE
Allow webpack alias definition

### ISSUE THREAD

**Issue by aach** [untool/untool#205]
I want to be able to define alias like

````javascript
const path = require('path');

module.exports = {
  "browsers": "last 1 Chrome versions",
  "alias": {
    "@": path.resolve(__dirname, 'src')
  },
  "locations": [
    "/",
    "/counter"
  ]
};
````
So, for example, we will be able to replace `../../../../../../../../components/Unicorn` by `@/components/Unicorn`

**Comment by dmbch** [comment]
Hej @aach, thanks a lot for bringing up this topic! I will answer in a bit more detail in #206.

**Comment by dmbch** [comment]
Closed in #214
